packages/frontend/src/common/components/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Box, Link, Paper } from "@mui/material";
+
+interface FooterProps {
+  isTransparent?: boolean;
```

Comment:
Why don't we hide single elements and not the complete footer?

Reply:
The footer is never hidden, it has to be included actively. But on the homepage the default theme style has to be overridden by setting the background color to transparent and the link color to black.